swagger.yaml

```diff
@@ -2085,6 +2126,24 @@ components:
         flextime:
           $ref: "#/components/schemas/Flextime"
 
+    UserContractedWorkWeek:
+      type: object
+      description: User work week and days worked in week, calculated from work days
+      properties:
+        contractedWeek:
+          type: array
+          description: Numeric representation of the user's contracted workdays in a week
+          items:
+            type: integer
+            minimum: 0
+            maximum: 6
+        contractedDaysCount:
+          type: integer
+          description: Total number of contracted workdays in the week
+      required:
+        - contractedWeek
+        - contractedDaysCount
+
```

Comment:
So as previous comment I suggested UserWorkWeekLength, let me know what you think.

Also do we really need to return this contractedWeek data? If so why?
 Can't we jsut return the number of days (contractedDayCount, which I would perhaps rename workingWeekLength).


I think typically the required is put before the properties for clarity.